src/modules/esl-event-listener/README.md

```diff
@@ -111,6 +111,27 @@ Here is the list of supported keys of `ESLEventDesriptor`:
   <u>Default Value:</u> `null`  
   <u>Description:</u> the CSS selector to filter event targets for event delegation mechanism.
 
+  ⚠ If you want to get the currently delegated event target, you can specify a new event type `DelegatedEvent<EventType>` for function and then the `$delegate` variable will contain the desired selector element.
+
```

Comment:
```suggestion
  ⚠ If you want to get the currently delegated event target, you can specify a new event type `DelegatedEvent<EventType>` for the function and then the `$delegate` variable will contain the desired selector element.

```

Reply:
at the suggestion of @ala-n, this part of the text was changed